fineract-provider/src/main/java/org/apache/fineract/useradministration/domain/AppUser.java

```diff
@@ -663,4 +663,8 @@ public boolean isSelfServiceUser() {
         return newAppUserClientMappings;
     }
 
+    @Override
+    public String toString() {
```

Comment:
Just curious, I'm not seeing this method being used anywhere, I wish to know why you wrote it

Reply:
Hello @xurror, if I am not wrong, it is explicitly used for debugging(if the test fails). 
Refer to this how it helped: #858  and https://issues.apache.org/jira/browse/FINERACT-828